Created a database relationship between user and post

khiebiggs opened the pull request:
Created a one to many relationship between the user and the post. First created a migration to create the structure in the table, then modfied the new post route to take in a token, which is then validated to return the users id, which is attached to the post object

khiebiggs committed:
Created a database relationship between user and post

Created a one to many relationship between the user and the post. First created a migration to create the structure in the table, then modfied the new post route to take in a token, which is then validated to return the users id, which is attached to the post object

khiebiggs committed:
Merge branch 'master' into posts

khiebiggs committed:
changed the type of object returns by new post routing

CLTPayne reviewed (changes requested):
In PostTest line 10, is 1L referring to a token? 
If so it's not so explicit. Could we adapt?

khiebiggs committed:
Made changes in response to code review
Code reviewers were unhappy with what "1L" referred to in the PostTest, so I created a Long called user_id to make the code more readable

CLTPayne reviewed (commented):
Great. Seems much more explicit now
